### PR TITLE
Export PageStorage V3 config into global config

### PR DESCRIPTION
**Pull request by jiaqizho**
### What problem does this PR solve?

Issue Number: ref #3594 

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Storages/Page/V3/WAL/WALReader.cpp [hunk 1]
```diff
@@ -122,27 +122,39 @@ WALStoreReader::findCheckpoint(LogFilenameSet && all_files)
...
+WALStoreReaderPtr WALStoreReader::create(
+    String storage_name,
+    FileProviderPtr & provider,
+    PSDiskDelegatorPtr & delegator,
+    WALRecoveryMode recovery_mode_,
+    const ReadLimiterPtr & read_limiter)
 {
     LogFilenameSet log_files = listAllFiles(delegator, Logger::get("WALStore", storage_name));
-    return create(storage_name, provider, std::move(log_files), read_limiter);
+    return create(std::move(storage_name), provider, std::move(log_files), recovery_mode_, read_limiter);
```
Comment (Lloyd-Pottiger) [inline dbms/src/Storages/Page/V3/WAL/WALReader.cpp:142]: do we need `std::move` here?

**Review by flowbehappy — APPROVED** [review]

File: dbms/src/Storages/Page/V3/WAL/WALReader.cpp (hunk 1 above)
Comment (jiaqizho) [reply dbms/src/Storages/Page/V3/WAL/WALReader.cpp:142]: can help reduce a string size.

**Review by Lloyd-Pottiger — APPROVED** [review]

**Comment by yibin87** [comment]
Unit test failed due to unstable gtest, just rerun and ignore it.

**Comment by jiaqizho** [comment]
> Unit test failed due to unstable gtest, just rerun and ignore it.

@yibin87 what should I do now?

**Pull request merged**